lib/net.js

```diff
@@ -1524,6 +1524,13 @@ Server.prototype.listen = function(...args) {
     backlog = options.backlog || backlogFromArgs;
     listenInCluster(this, pipeName, -1, -1,
                     backlog, undefined, options.exclusive);
+    let mode = 0;
+    if (!!options.readable_all)
```

Comment:
Since these are expected to be booleans, you could do `options.readableAll === true`.